src/components/editions/article.tsx

```diff
@@ -27,7 +37,7 @@ const bodyStyles = css`
 
 	${from.phablet} {
 		margin-left: ${remSpace[24]};
-		width: ${wideContentWidth}px;
+		width: ${editionsArticleWidth}px;
```

Comment:
Would we want to do this with `rem`s? What happens if you zoom (`Cmd +`)?

Reply:
So I think this behaves as I'd imagined, ie it zooms ok. Have changed to REM though if that's the preferred approach.